content/doc/book/pipeline/syntax.adoc

```diff
@@ -717,6 +720,115 @@ pipeline {
 <1> The tool name must be pre-configured in Jenkins under *Manage Jenkins* ->
 *Global Tool Configuration*.
 
+==== stage options
+
+The `options` directive for a `stage` is similar to the `options` directive at
+the root of the Pipeline. However, the `stage`-level `options` can only contain
+steps like `retry`, `timeout`, or `timestamps`, or Declarative options that are
+relevant to a `stage`, like `skipDefaultCheckout`.
+
+Steps in a `stage`'s `options` are invoked before entering the `stage`'s `agent`
```

Comment:
single backtick doesn't work mid-word.  There's a syntax that does, but it is problematic.  See https://issues.jenkins-ci.org/browse/WEBSITE-428

Suggest changing to:
```
Inside a `stage`, the steps an `options` section are invoked before entering the `agent`
```

Also, search and replace the rest of your change for this same problem.

Reply:
Looks like that's the only line.